### DISCUSSION TITLE
BirdNet on Libre Renegade or Orange Pi 5?

### DISCUSSION THREAD

**Discussion by bcardi0427** [mcguirepr89/BirdNET-Pi#688] [General]
Has anyone tried to get BirdNet on either of these SBC's? Amazon has the Renegade with 4G Ram for $50 in stock. It says it can run the following os besides Debian and Ubuntu:
Raspbian 10 Buster Lite and Desktop armhf
Raspbian 11 Bullseye Lite and Desktop armhf
Raspbian 11 Bullseye Lite and Desktop arm64
I'm thinking of picking one up and trying it out. The Orange Pi has a Raspberry OS that it runs but I'm not sure how good that os is. Anyone know anything about either of these?

**Comment by arduionoGP** [comment]
I have it running on both the Libre potato and the Libre renegade version with 4GB.
